client/utils/auth.py

```diff
@@ -169,53 +169,60 @@ def perform_oauth(assignment, code_fn):
     update_storage(access_token, expires_in, refresh_token)
     return access_token
 
-def authenticate(assignment, force=False):
+def server_url(cmd_args):
```

Comment:
Actually - `cmd_args` could also be an `argparse.Namespace` object and they should be compatible. I'll keep it as is for now